test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/InstallCommandTests.cs

```diff
@@ -255,5 +257,51 @@ public static CommandRunnerResult RunInstall(string nugetExe, SimpleTestPathCont
 
             return r;
         }
+
+        [Fact]
+        public async Task Install_WithHttpSource_Warns()
+        {
+            // Arrange
+            using var pathContext = new SimpleTestPathContext();
+            // Set up solution, project, and packages
+            var solution = new SimpleTestSolutionContext(pathContext.SolutionRoot);
+            var packageA = new SimpleTestPackageContext("a", "1.0.0");
+            await SimpleTestPackageUtility.CreateFolderFeedV3Async(pathContext.PackageSource, packageA);
+            var packageAPath = Path.Combine(pathContext.PackageSource, "a", "1.0.0", "a.1.0.0.nupkg");
+
+            pathContext.Settings.AddSource("http-feed", "http://api.source/api/v2");
+            pathContext.Settings.AddSource("https-feed", "https://api.source/index.json");
+
+            var net461 = NuGetFramework.Parse("net461");
+
+            var projectA = SimpleTestProjectContext.CreateLegacyPackageReference(
```

Comment:
Shouldn't use a LegacyPackageReference project, but just a framework project here.

Reply:
I found an example in test code on how to create a framework project.
``` cs
var projectA = new SimpleTestProjectContext(
                    "a",
                    ProjectStyle.PackagesConfig,
                    pathContext.SolutionRoot);
                projectA.Frameworks.Add(new SimpleTestProjectFrameworkContext(net461));
```